### PR TITLE
FISH-7016: adding fix when ports are used from another instance

### PR DESCRIPTION
**Pull request by breakponchito**
Fix for zombie instance when ports are used from another instance
## Description

This is a fix for a bug reported when trying to start an instance from a deployment group trying to use ports used for another available running instance like payara micro

### Testing Performed

Manual testing, now the error is showed on the logs when trying to start the instance:

https://user-images.githubusercontent.com/279375/223603704-657331b5-e86e-4c8c-a0db-033184b336be.mp4

On logs now we can see the following message:

`
Command start-local-instance failed.

Port 28080 is in use

To complete this operation run the following command locally on host localhost from the Payara Server install location C:\java\projects\Payara\appserver\distributions\payara\target\stage\payara6:

 lib/nadmin  start-local-instance --node localhost-domain1 --sync normal --timeout 600 instance1

The command start-instance failed for: instance1
Command start-deployment-group failed.
`

### Testing Environment

Windows 11, azul jdk 11, maven 3.8.3

### CODE REVIEW

**Comment by breakponchito** [comment]
Jenkins test please

**Comment by breakponchito** [comment]
Jenkins test please

File: nucleus/admin/server-mgmt/src/main/java/com/sun/enterprise/admin/servermgmt/cli/StartServerHelper.java [hunk 1]
```diff
@@ -296,9 +303,11 @@ private void waitForParentToDie() throws CommandException {
 
     private boolean checkPorts() {
         String err = adminPortInUse();
+        err = validateAdditionalPortsForConnection();
 
         if (err != null) {
-            logger.warning(err);
+            //incrementing level of severity when ports are used
```
Comment (Pandrex247) [inline nucleus/admin/server-mgmt/src/main/java/com/sun/enterprise/admin/servermgmt/cli/StartServerHelper.java:309]: Don't think this comment is necessary
```suggestion
```

**Review by Pandrex247 — APPROVED** [review]

**Comment by breakponchito** [comment]
Jenkins test please

File: nucleus/admin/server-mgmt/src/main/java/com/sun/enterprise/admin/servermgmt/cli/StartServerHelper.java (hunk 1 above)
Comment (breakponchito) [reply nucleus/admin/server-mgmt/src/main/java/com/sun/enterprise/admin/servermgmt/cli/StartServerHelper.java]: already done

**Pull request merged**